test/fixtures/full/variables.tf

```diff
@@ -0,0 +1,8 @@
+variable "dataset_id" {}
+variable "dataset_name" {}
+variable "description" {}
+variable "expiration" {}
+variable "project_id" {}
+variable "table_id" {}
+variable "time_partitioning" {}
+variable "schema_file" {}
```

Comment:
Why does the test/fixtures variables.tf have more variables defined than the root module? Apologies for not understanding the syntax.